[TH] Add Beatmap Nominators (#4480)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/th.md

---
tags:
  - BN
  - BNG
---

# Beatmap Nominators

> *ดูสิ่งนี้ด้วย: [กฎของ Beatmap Nominators](Rules), [ความคาดหวังสำหรับ Beatmap Nominators](Expectations), [ข้อมูลทั่วไปสำหรับ Beatmap Nominators](General_Information), และ [การมาเป็น Beatmap Nominators](Becoming_a_Beatmap_Nominator)*

Reviewer: ```suggestion
*ดูสิ่งนี้ด้วย: [กฎของ Beatmap Nominator](Rules), [ความคาดหวังสำหรับ Beatmap Nominators](Expectations), [ข้อมูลทั่วไปสำหรับ Beatmap Nominators](General_Information), และ [การมาเป็น Beatmap Nominator](Becoming_a_Beatmap_Nominator)*
```
From original article

กลุ่ม **Beatmap Nominators** (***BN***) คือกลุ่มที่รับผิดชอบเกี่ยวกับการ[เสนอบีทแมพ](/wiki/Beatmap_ranking_procedure) โดยพวกเขามีชื่อสีม่วงในฟอรั่มและในที่พูดคุยบีทแมพ

## หน้าที่และความรับผิดชอบ

> สมาชิกของกลุ่ม Beatmap Nominators รับผิดชอบหลัก ๆ ในการ[วิจารณ์บีทแมพ](/wiki/Modding)ที่อยู่ในสถานะ [pending](/wiki/Beatmaps#work-in-progress-and-pending) และอนุมัติไปในการแรงค์ โดย Beatmap Nominators เป็นผู้คนที่สามารถถามได้เกี่ยวกับคำแนะนำ หรือคำขอต่าง ๆ เกี่ยวกับเรื่องใด ๆ ก็ได้ที่เกี่ยวกับการแมปหรือการวิจารณ์บีทแมพ Beatmap Nominators คนใดก็ได้สามารถยับยั้งการแรงค์บีทแมพ ([veto](Beatmap_Veto)) หากเขารู้สึกว่าแมปที่ถูกเสนอมีปัญหา หรือมีข้อกังวลที่ต้องถูกแก้ ที่ไม่ได้ละเมิดกฎการแรงค์แบบตรง ๆ

Reviewer: ```suggestion
สมาชิกของกลุ่ม Beatmap Nominators รับผิดชอบหลัก ๆ ในการ[วิจารณ์บีทแมพ](/wiki/Modding)ที่อยู่ในสถานะ [pending](/wiki/Beatmaps#work-in-progress-and-pending) และอนุมัติไปในการแรงค์ โดย Beatmap Nominators เป็นผู้คนที่สามารถถามได้เกี่ยวกับคำแนะนำ หรือคำขอต่าง ๆ เกี่ยวกับเรื่องใด ๆ ก็ได้ที่เกี่ยวกับการแมพหรือการวิจารณ์บีทแมพ Beatmap Nominator คนใดก็ได้สามารถยับยั้งการแรงค์บีทแมพ ([veto](Beatmap_Veto)) หากเขารู้สึกว่าแมพที่ถูกเสนอมีปัญหา หรือมีข้อกังวลที่ต้องถูกแก้ ที่ไม่ได้ละเมิดกฎการแรงค์แบบตรง ๆ
```

สมาชิกของกลุ่ม Beatmap Nominators ได้ถูกแบ่งเป็นสองกลุ่ม: สมาชิกเต็มรูปแบบ และสมาชิกภาคทัณฑ์

### Beatmap Nominators เต็มรูปแบบ

นอกเหนือจากการวิจารณ์บีทแมพและแรงค์โดยธรรมดาแล้ว สมาชิกเต็มรูปแบบของกลุ่ม Beatmap Nominators มีความรับผิดชอบในการปลด (disqualify) บีทแมพในหมวด qualified ที่มีปัญหา โดย Beatmap Nominators เต็มรูปแบบสามารถมีส่วนร่วมในการประเมินใบสมัครเข้าร่วมกลุ่ม Beatmap Nominators หากพวกเขามีความสนใจในการเข้าร่วม Nomination Assessment Team ในอนาคต

ประมาณทุก ๆ สามเดือน สมาชิก Beatmap Nominators เต็มรูปแบบทุกคน จะถูกประเมินพฤติกรรม กิจกรรม และคุณภาพในการเสนอ โดย [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) หากการเสนอของพวกเขาถูกรีเซ็ตบ่อย ๆ อย่างมีสาเหตุ หรือหากพฤติกรรมของพวกเขาไม่สามารถถูกรับได้ พวกเขาอาจจะถูกย้ายไปกลุ่มสมาชิกภาคทัณฑ์

สมาชิกเต็มรูปแบบของกลุ่ม Beatmap Nominators จะไม่โดนภาคทัณฑ์หากกิจกรรมของพวกเขาน้อยเกินไป แต่พวกเขาจะโดนปลดจากกลุ่มนี้หากกิจกรรมของเขาไม่ดีขึ้น

### Beatmap Nominators ภาคทัณฑ์

> การภาคทัณฑ์ถูกใช้เพื่อการสังเกตการณ์ Beatmap Nominators ใหม่ หรือที่มีพฤติกรรมน่ากังวล อย่างใกล้ชิด โดยหน้าที่หลักของ Beatmap Nominators ภาคทัณฑ์คือการสร้างรากฐานในการเป็น Beatmap Nominators โดยการเสนอบีทแมพ (อย่างเดียว) ในกระบวนการแรงค์ โดยในการเลื่อนขั้นบีทแมพ Beatmap Nominators เต็มรูปแบบต้องมีส่วนร่วมอย่างน้อยหนึ่งคน ดังนั้น Beatmap Nominators ภาคทัณฑ์จึงไม่สามารถเป็นคนสุดท้ายที่เสนอบีทแมพที่ได้รับการเสนอจาก Beatmap Nominators ภาคทัณฑ์คนอื่น สำหรับเกมโหมดนั้น ๆ

Reviewer: ```suggestion
การภาคทัณฑ์ถูกใช้เพื่อการสังเกตการณ์ Beatmap Nominators ใหม่ หรือที่มีพฤติกรรมน่ากังวล อย่างใกล้ชิด โดยหน้าที่หลักของ Beatmap Nominators ภาคทัณฑ์คือการสร้างรากฐานในการเป็น Beatmap Nominator โดยการเสนอบีทแมพ (อย่างเดียว) ในกระบวนการแรงค์ โดยในการเลื่อนขั้นบีทแมพ Beatmap Nominators เต็มรูปแบบต้องมีส่วนร่วมอย่างน้อยหนึ่งคน ดังนั้น Beatmap Nominators ภาคทัณฑ์จึงไม่สามารถเป็นคนสุดท้ายที่เสนอบีทแมพที่ได้รับการเสนอจาก Beatmap Nominators ภาคทัณฑ์คนอื่น สำหรับเกมโหมดนั้น ๆ
```
